view/theme/frio/templates/admin/logs/view.tpl

```diff
@@ -65,11 +65,14 @@
 					data-data="{{$row->data}}" data-source="{{$row->source}}">
 					<td>{{$row->date}}</td>
 					<td class="
-						{{if $row->level == "CRITICAL"}}bg-danger
-						{{elseif $row->level == "ERROR"}}bg-danger
+						{{if $row->level == "EMERGENCY"}}bg-danger
+						{{elseif $row->level == "ALERT"}}bg-danger
+						{{elseif $row->level == "CRITICAL"}}bg-warning
+						{{elseif $row->level == "ERROR"}}bg-warning
```

Comment:
We don't really distinguish between these 4 log levels in practice, so I would set them all to `bg-danger`. On the other hand, warnings can often be ignored.

Reply:
This "on the other hand" confuses me. Should I change it?

Reply:
What I'm saying is that it is counter-productive to "hide" error and critical messages behind the Warning color which doesn't necessarily prompts an admin to react to these messages.

Reply:
Changed